src/assets/translations/en/main.json

```diff
@@ -2585,7 +2586,25 @@
   "markets": {
     "recommended": "Recommended",
     "watchlist": "My Watchlist",
-    "categories": "Categories",
-    "marketsBody": "Nunc vel efficitur ligula, vel sagittis est. Morbi et sapien a ligula venenatis consequat."
+    "categoriesTabTitle": "Categories",
+    "marketsBody": "Nunc vel efficitur ligula, vel sagittis est. Morbi et sapien a ligula venenatis consequat.",
+    "categories": {
+      "oneClickDefiAssets": {
+        "title": "One Click DeFi Assets"
+      },
+      "thorchainDefi": {
+        "title": "THORChain DeFi"
+      },
+      "trending": {
+        "title": "Trending",
+        "subtitle": "These are top assets that have jumped 10% or more"
```

Comment:
[suggestion] We could parameterize this e.g `"These are top assets that have jumped %{percentage} or more"`

Reply:
5fa241b41a